specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -992,6 +1085,54 @@
       "additionalProperties": true,
       "description": "A single bucket of a facet query result. Reports the number of documents with a field value falling within a particular range or having a particular value or interval."
     },
+    "AnswerResult": {
+      "properties": {
+        "score": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The score value represents how relevant the answer is to the the query relative to other answers returned for the query."
+        },
+        "key": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The key of the document the answer was extracted from.",
+          "x-ms-client-name": "Key"
```

Comment:
1. You don't have this on `score`.
2. You don't really need it when it's the same as the property. Our generators will case as appropriate for the language.

Reply:
For 2, this applies everywhere else in this PR as well.